### ISSUE TITLE
FileNotFoundError: [Errno 2] No such file or directory: 'README.md'

### ISSUE THREAD

**Issue by jatindergit** [vishalanandl177/DRF-API-Logger#5]
Please update your setup.py in https://files.pythonhosted.org/packages/e0/26/6c903af8943b1f91584401d122e8d62ca4d2b778245f2afaee37a12484db/drf_api_logger-1.0.3.tar.gz#sha256=3b0dab7b0ccd9035425a18d0599d7f0a56c28a17108a02aa667e8a61fb50efdc

Error
    ```
ERROR: Command errored out with exit status 1:
     command: /usr/local/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-kuuj6y0c/drf-api-logger_49ecd2f8baa1419691e19dcc415c23a3/setup.py'"'"'; __file__='"'"'/tmp/pip-install-kuuj6y0c/drf-api-logger_49ecd2f8baa1419691e19dcc415c23a3/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-pu29f_p6
         cwd: /tmp/pip-install-kuuj6y0c/drf-api-logger_49ecd2f8baa1419691e19dcc415c23a3/
    Complete output (7 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-kuuj6y0c/drf-api-logger_49ecd2f8baa1419691e19dcc415c23a3/setup.py", line 16, in <module>
        long_description=get_long_desc(),
      File "/tmp/pip-install-kuuj6y0c/drf-api-logger_49ecd2f8baa1419691e19dcc415c23a3/setup.py", line 5, in get_long_desc
        with open("README.md", "r") as fh:
    FileNotFoundError: [Errno 2] No such file or directory: 'README.md'
    ----------------------------------------
WARNING: Discarding https://files.pythonhosted.org/packages/e0/26/6c903af8943b1f91584401d122e8d62ca4d2b778245f2afaee37a12484db/drf_api_logger-1.0.3.tar.gz#sha256=3b0dab7b0ccd9035425a18d0599d7f0a56c28a17108a02aa667e8a61fb50efdc (from https://pypi.org/simple/drf-api-logger/) (requires-python:>=3.5). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
ERROR: Could not find a version that satisfies the requirement drf-api-logger==1.0.3
ERROR: No matching distribution found for drf-api-logger==1.0.3
```

you have spelling mistake in reading README.md file in setup.py

**Comment by vishalanandl177** [comment]
Please try to uninstall and install it again.
Let me know if the issue still persists.

**Comment by jatindergit** [comment]
Still same error

**Comment by vishalanandl177** [comment]
Please comment here python version, OS, drf version.

**Comment by jatindergit** [comment]
Python 3.8.5 (v3.8.5:580fbb018f, Jul 20 2020, 12:11:27) 
[Clang 6.0 (clang-600.0.57)] on darwin
djangorestframework       3.12.2
MAC OS

**Comment by marzique** [comment]
Same error while trying to pip install

drf 3.12.2
python 3.6.12
Ubuntu 20.04.1 LTS via Windows 10 WSL

**Comment by danishahmed90** [comment]
I am also facing the same error. I think this is happened after some recent commits.

**Comment by vishalanandl177** [comment]
Thanks for letting me know. 
It will get fixed in 24 hours.

**Comment by vishalanandl177** [comment]
A new version is released 1.0.4 with the fixes.
